packages/components/format-date/src/format-date.ts

```diff
@@ -0,0 +1,100 @@
+import { LocaleMixin } from '@sl-design-system/shared';
+import { LitElement, type TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { formatDateTime } from './format-date-time';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-format-date': FormatDate;
+  }
+}
+
+/**
+ * A format date component for formatting date and time.
+ *
+ * @slot default - A place for the fallback when there is no valid date/time applied.
+ */
+export class FormatDate extends LocaleMixin(LitElement) {
+  #date?: Date = new Date();
```

Comment:
Won't this cause the fallback never to be rendered? Unless you first set the `date` property to null/undefined/something that is not a date?

Reply:
My initial idea with `<dna-format-date>` is that if the developer does not set a date, or it is undefined, then it by default shows the slot content.